syntax = "proto3";

import "ColorMap.proto";

package ElVis.Serialization;

message ColorMapperModule {
  ColorMap color_map = 1;
  uint32 size = 2;
}
